Makefile

```diff
@@ -32,4 +32,6 @@ GO=go
 GOFLAGS=""
 
 test:
```

Comment:
As we discussed earlier, this target could be split into a target for the Go tests and a target for the shell tests, so they could be both triggered independently, if needed. The `test` target could call both of them.

This can be done later, just mentioning it here so it does not get forgotten.

Reply:
I agree, this will be addressed in the next PRs.

Reply:
Since this will be executing two separate frameworks, should we go head a make a `test-go` target to separate them now